tests/meta/upload/gisaid/conftest.py

```diff
@@ -81,16 +80,26 @@ def gisaid_case_id():
     return "gisaid_case"
 
 
+@pytest.fixture(name="gisaid_file_name")
```

Comment:
can this be saved to Path(tmpdir, "test.fasta")? then tmpdir is cleaned after pytest is done
even though this is rmoved once not used, still its kind of weird to save and delete file from repo base every time

Reply:
Haha, yes that's exactly what I did first

Reply:
But there is a test that checks for the filename in a file, so it is hardcoded there. And the tests will fail